doc/developers/contributing.rst

```diff
@@ -1512,6 +1512,10 @@ stateless
     whether the estimator needs access to data for fitting. Even though
     an estimator is stateless, it might still need a call to ``fit`` for initialization.
 
+requires_fit
+    whether the estimator requires to be fitted before to call one of
+    ``transform``, ``predict``, ``predict_proba``, or ``decision_function``.
```

Comment:
The method names (here and below) could be using single backticks to enable links to the glossary